readthedocs/templates/errors/404/include_tips.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
+
+<h3>{% trans "Are you the project owner?" %}</h3>
+
+<p>{% trans "Here are some tips to address 404 errors:" %}</p>
+
+<ul>
+  <li>{% trans "Use custom 404 pages:" %} <a href="https://docs.readthedocs.io/en/latest/hosting.html#custom-not-found-404-pages">{% trans "Read more" %}»</a></li>
+  <li>{% trans "Create redirects when you move contents:" %} <a href="https://docs.readthedocs.io/en/stable/user-defined-redirects.html">{% trans "Read more" %}»</a></li>
+  <li>{% trans "Use rediraffe to manage redirects in Sphinx:" %} <a href="https://pypi.org/project/sphinxext-rediraffe/">{% trans "Read more" %}»</a></li>
```

Comment:
I'm not sure we want to recommend rediraffe here since it uses Javascript/meta tags for redirects.

Reply:
Yes, I wanted to discuss precisely these suggestions earlier :100: 

In your experience, what are some commonly missed 404 options?

Reply:
I'm not sure to understand the question 😅 

BTW, the other 2 suggestions are 💯 to me: custom 404 and our own redirects.

This was a project we will want to recommend in the future once it gets stable: https://github.com/nextstrain/readthedocs-cli. It allows you to sync redirects on RTD with a YAML file.

Reply:
Totally agree that Rediraffe is too hacky a solution.

Exciting to see where nexstrain's readthedocs-cli is going :+1: